### DISCUSSION TITLE
DDS Router Docker image not compatible with arm64 systems

### DISCUSSION THREAD

**Discussion by Vicidel** [eProsima/DDS-Router#344] [Ideas]
Hello all

I want to run the FastDDS DDS Router Docker image on an arm64 system, but the image gives this error:
```
exec /bin/bash: exec format error
```

I guess you are only building the image for amd64 systems. Would it be possible to support it in the future?
Best, Victor

**Comment by juanlofer-eprosima** [comment]
Hi @Vicidel

Thank you for reaching out. Unfortunately, there is currently no DDS-Router image for arm64 systems.

However, you may instead pull [Vulcanexus](https://vulcanexus.org/) Cloud Docker image from Docker Hub, which includes DDS-Router and is available for this platform. For this, it suffices to execute: 
`docker run --rm -it --net=host -v /path/to/config.yaml:/DDS_ROUTER_CONFIGURATION.yaml eprosima/vulcanexus:humble-cloud ddsrouter`

Best regards

**Comment by Vicidel** [comment]
Thanks for the command, I'll try it. If you directly support `arm64` it would help my setup :)
